installer/installer.py

```diff
@@ -428,20 +434,27 @@ def post_install(self):
 
         # Setting up the external_address
         if self.interviewer.external_address:
-            self.exec(f"sudo su -c 'cheqd-noded configure p2p external-address {self.interviewer.external_address}:{self.interviewer.p2p_port}' {DEFAULT_CHEQD_USER}")
-
+            external_address_search_text='external_address = ""'
+            external_address_replace_text='external_address = "{}:{}"'.format(self.interviewer.external_address, self.interviewer.p2p_port)
+            search_and_replace(external_address_search_text, external_address_replace_text, os.path.join(self.cheqd_config_dir, "config.toml"))
```

Comment:
If the user _doesn't_ enter any value here, it should execute line 968-970 and use a live lookup to figure out the IP address of the server and set it. It doesn't seem to do that - or at least, it's not writing it to the config.toml even if it has grabbed that DNS address.

Reply:
Okay, I see. I tried the command inside VM myself, the problem seems to be not being able to grab it form `stdout` and set it the value to `external_address`. I am looking into it. 👍🏾